test/unit/specs/methods.spec.js

```diff
@@ -69,10 +70,131 @@ describe('ti.map', function () {
 			// win.add(view);
 			// win.open();
 		});
+		if (!ANDROID) {
+		// TiMapImageOverlayProxy
+			describe('setImage', () => {
+				it('to set image on Map',  () => {
+					var fileSystem = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'images', 'flower.jpg.keep');
+					var newBlob = fileSystem.read();
+
+					expect(Map.setImage(newBlob)).toEqual(jasmine.any(Function));
+				});
+			});
 
-		// TDOO: Test actually adding an annotation to a view
+			describe('setBoundsCoordinate', () => {
+				it('to set bound coordinates',  () => {
+					var fileSystem = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'images', 'flower.jpg.keep');
+					var newBlob = fileSystem.read();
+
+					expect(Map.setBoundsCoordinate(newBlob)).toEqual(jasmine.any(Function));
+				});
+			});
+			// TiMapCircleProxy
+			describe('setCenter', () => {
+				it('to set the center',  () => {
+					var latLongData = ({ latitude: -33.86365, longitude: 151.21689 });
+
+					expect(Map.setCenter(latLongData)).toEqual(jasmine.any(Function));
+				});
+			});
+
+			describe('setFillColor', () => {
+				it('to set the fill color',  () => {
+					var fillColor = ({ fillColor: '#BFFFE725' });
+
+					expect(Map.setCenter(fillColor)).toEqual(jasmine.any(Function));
+				});
+			});
+
+			describe('setStrokeColor', () => {
+				it('to set the stroke Color',  () => {
+					var strokeColor = ({ strokeColor: '#D2BE1F' });
+
+					expect(Map.setCenter(strokeColor)).toEqual(jasmine.any(Function));
+				});
+			});
+
+			describe('setStrokeWidth', () => {
+				it('to set the stroke width',  () => {
+					var strokeWidth = ({ strokeWidth: 2 });
+
+					expect(Map.setCenter(strokeWidth)).toEqual(jasmine.any(Function));
+				});
+			});
+
+			describe('setOpacity', () => {
+				it('to set the opacity',  () => {
+					var opacity = ({ opacity: 75 });
+
+					expect(Map.setCenter(opacity)).toEqual(jasmine.any(Function));
+				});
+			});
+			// TiMapPolygonProxy
+			describe('setPoints', () => {
+				it('to set the points',  () => {
+					var points = ([
+						[ 151.228290, -33.857280 ],
+						[ 151.224428, -33.855427 ],
+						[ 151.224170, -33.858991 ]
+					]);
+
+					expect(Map.setCenter(points)).toEqual(jasmine.any(Function));
+				});
+			});
+
+			describe('setPattern', () => {
+				it('to set the pattern',  () => {
+					var pattern = ({
+						type: Map.POLYLINE_PATTERN_DASHED,
+						gapLength: 25,
+						dashLength: 50
+					});
+
+					expect(Map.setCenter(pattern)).toEqual(jasmine.any(Function));
+				});
+			});
+			// TiMapSnapShotterProxy
+			describe('setRegion', () => {
```

Comment:
@anil-shukla-axway  why did you add Snapshotterproxy test case in this pr. Snapshotterproxy is covered in different ticket so it should have its own pr.